### PR TITLE
feat(web-analytics): Add session attribution explorer

### PR DESCRIPTION
**Pull request by robbie-c**
## Problem

Many customer support requests are about the session attribution. I often send customers this HogQL query or run it for them, I figured it would make sense to create a page for it.

## Changes

Here's what the explorer looks like:
Screenshot 2024-07-04 at 13 36 42

I added a link to the channel tooltip on web analytics:
Screenshot 2024-07-03 at 19 58 59

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

Run manually on localhost, it's v similar to the query that I've been sending customers

### CODE REVIEW

File: posthog/api/test/test_session.py [hunk 1]
```diff
@@ -85,8 +85,8 @@ def test_list_channel_type_values(self):
             "Organic Search",
             "Organic Shopping",
             "Organic Video",
-            "Other",
-            "Paid Other",
+            "Unknown",
+            "Paid Unknown",
```
Comment (robbie-c) [inline posthog/api/test/test_session.py:89]: Unrelated bug that I noticed while doing this, and fixed

File: frontend/src/scenes/web-analytics/SessionDebugger/sessionAttributionExplorerLogic.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    actionToUrl(({ values }) => {
+        const stateToUrl = (): string => {
+            const { filters } = values
+
+            const urlParams = new URLSearchParams()
+            if (filters.length > 0) {
+                urlParams.set('filters', JSON.stringify(filters))
+            }
+
+            return `${urls.sessionAttributionExplorer()}?${urlParams.toString()}`
```
Comment (pauldambra) [inline frontend/src/scenes/web-analytics/SessionDebugger/sessionAttributionExplorerLogic.ts:83]: i'm sure there's a safer way to do this...

Comment (pauldambra) [reply frontend/src/scenes/web-analytics/SessionDebugger/sessionAttributionExplorerLogic.ts:83]: yep, look at examples that look like

```
        hideNewDashboardModal: () => {
            const hashParams = router.values.hashParams
            delete hashParams['newDashboard']
            return [router.values.location.pathname, router.values.searchParams, hashParams]
        },
```

Comment (pauldambra) [reply frontend/src/scenes/web-analytics/SessionDebugger/sessionAttributionExplorerLogic.ts:83]: (means you can avoid destroying parts of the url / messing around with string concatentation

**Review by pauldambra — APPROVED** [review]
bit tricky to find - could be a tab in data management or a kebab menu at the top of web analytics - but progress over perfection and all that jazz

File: frontend/src/scenes/web-analytics/SessionDebugger/sessionAttributionExplorerLogic.ts (hunk 2 above)
Comment (robbie-c) [reply frontend/src/scenes/web-analytics/SessionDebugger/sessionAttributionExplorerLogic.ts]: Cool! didn't know it was possible, have updated it :)

**Pull request merged**